sdks/python/apache_beam/metrics/monitoring_infos.py

```diff
@@ -56,6 +56,8 @@
     [USER_COUNTER_URN, USER_DISTRIBUTION_URN, USER_GAUGE_URN])
 WORK_REMAINING_URN = common_urns.monitoring_info_specs.WORK_REMAINING.spec.urn
 WORK_COMPLETED_URN = common_urns.monitoring_info_specs.WORK_COMPLETED.spec.urn
+DATA_CHANNEL_READ_INDEX = (
+    common_urns.monitoring_info_specs.DATA_CHANNEL_READ_INDEX.urn)
```

Comment:
Looks like you are missing `spec` here.

Reply:
good find

Reply:
Yeah, I just found that myself... Not sure why it has a urn and a spec.urn.